pkgs/tools/networking/gping/default.nix

```
@@ -9,21 +9,25 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "gping";
-  version = "1.13.1";
+  version = "1.14.0";
 
   src = fetchFromGitHub {
     owner = "orf";
     repo = "gping";
     rev = "gping-v${version}";
-    hash = "sha256-EkoOHyHYcbyqtT1zCq0kmXND1eSADE7QD3QQ01RJtvM=";
+    hash = "sha256-ReP+s2p0X39LVvl3/QF7fsYkU+OvsQyMhyuH8v4HuVU=";
   };
 
-  cargoHash = "sha256-iDB3ZIlSLEBf+nSxLeQcE93nqMjH29w+z7kwCNksuSk=";
+  cargoHash = "sha256-FTiNxCoEe/iMz68F1CpJHypgrhn4WwwWowuN9I1rl6E=";
 
   buildInputs = lib.optionals stdenv.isDarwin [ libiconv Security ];
 
   nativeCheckInputs = lib.optionals stdenv.isLinux [ iputils ];
 
+  postInstall = ''
+    install -D gping.1 -t $out/share/man/man1
```

please use `installManPage` instead, you would also need to add `installShellFiles` to `nativeBuildInputs`

```suggestion
    installManPage gping.1
```

---

Thanks for the review, I tried to incorporate the changes you suggested.